src/subarraynew.jl

```diff
@@ -0,0 +1,36 @@
+typealias NullableSubArray{T,N,P<:NullableArray,IV,LD} SubArray{T,N,P,IV,LD}
+
+@inline function Base.isnull(V::NullableSubArray, I::Real...)
+    @boundscheck checkbounds(V, I...)
+    @inbounds return V.parent.isnull[Base.reindex(V, V.indexes, Base.to_indexes(I...))...]
```

Comment:
Presumably the commits will be squashed, removing the the addiction and
subtraction of that feature?  I believe this is what @naliman was referring
to.

On Monday, April 4, 2016, Scott P. Jones <notifications@github.com> wrote:

> In src/subarraynew.jl
> <https://github.com/JuliaStats/NullableArrays.jl/pull/101#discussion_r58371197>
> :
>
> > @@ -0,0 +1,36 @@
> > +typealias NullableSubArray{T,N,P<:NullableArray,IV,LD} SubArray{T,N,P,IV,LD}
> > +
> > +@inline function Base.isnull(V::NullableSubArray, I::Real...)
> > +    @boundscheck checkbounds(V, I...)
> > +    @inbounds return V.parent.isnull[Base.reindex(V, V.indexes, Base.to_indexes(I...))...]
>
> @nalimilan <https://github.com/nalimilan> They were necessary to have the
> subscripts take Real, which would be more consistent with the code in
> Base/subarray.jl, however, I removed them based on @mbauman
> <https://github.com/mbauman> review comment above.
> I do feel that it's incorrect to have these act differently than plain
> SubArrays, however, that could be fixed in a separate PR if people agree.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaStats/NullableArrays.jl/pull/101/files/a2a4740c12a904f5bd11672d1190ef0eca59b02f..6a849974f7af5b75df58727dbe9f22dc5b978960#r58371197>
>